Gemini/src/Migrations/Version20180530031926.php

```diff
@@ -12,20 +12,37 @@
 {
     public function up(Schema $schema)
     {
-        // this up() migration is auto-generated, please modify it to your needs
-        $this->abortIf(
-            $this->connection->getDatabasePlatform()->getName() !== 'mysql',
-            'Migration can only be executed safely on \'mysql\'.'
-        );
```

Comment:
Oh, last comment. This aborIf should stay in an extra "else" at the bottom, we don't want that folks with Oracle or Ms SQL Server have a false notion that their tables were created because they did not get any feedback from running the script

Reply:
And say... Migration can only be executed safely on \'mysql\'  or \'postgres\'

Reply:
and without the `this->connection->getDatabasePlatform()->getName() !==  because we already checked for that in the if and the ifelse right? so basically a $this->abortIf(TRUE, $message)?

Reply:
Added. (See below.)